Description: Remove bindgen feature
 We remove the libraw-rs-sys/bindgen feature because we always
 use bindgen. This is because we link towards the libraw provided by the
 system.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -27,6 +27,3 @@
 name = "libraw"
 [dependencies.libraw-rs-sys]
 version = "0.0.4"
-
-[features]
-bindgen = ["libraw-rs-sys/bindgen"]
